### PR TITLE
devops implementation

### PR DESCRIPTION
**Pull request by seenu433**
The PR includes the below

- source code for the function
- pipelines to deploy
  - Function
  - APIs
- readme.md with instruction

Change made to the createvm.bicep to change the agent pool name from Default to apim-cs

### CODE REVIEW

**Comment by ahmedsza** [comment]
Hi @seenu433 . What do you propose to change the name of the pool. The default pool already exists. Does this change mean that the new pool with this name needs to already exist? I assume it does - and if so how will you handle it. I would recommend keeping the default pool - as you are likely to have failures if customers do not create the pool with this name.

**Comment by seenu433** [comment]
@ahmedsza - The agent was not showing up in the default pool for me so had to use a different name. Please test it with default and let me know if it works. You are right that the pool should be created before running the scripts and the instructions are provided in readme.md

**Comment by petemessina** [comment]
@seenu433 The .vscode folder should be in the gitignore
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
!.vscode/*.code-snippets

File: deployment/bicep/shared/createvmwindows.bicep [hunk 1]
```diff
@@ -34,7 +34,7 @@ param accountName string=''
 param personalAccessToken string=''
 
 @description('The name Azure DevOps or GitHub pool for this build agent to join. Use \'Default\' if you don\'t have a separate pool.')
-param poolName string = 'Default'
+param poolName string = 'apim-cs'
```
Comment (petemessina) [inline deployment/bicep/shared/createvmwindows.bicep:37]: Also, we should revert this back to Default

File: src/readme.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Steps to deploy the function and APIs
+
+- [Import](https://docs.microsoft.com/en-us/azure/devops/repos/git/import-git-repository?view=azure-devops) this repo to an Azure DevOps Repo
+- Create two [ARM service connections](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/connect-to-azure?view=azure-devops) each scoped to the apim resource group and the fucntion app resource group
+- Create a new [Agent pool](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/pools-queues?view=azure-devops&tabs=yaml%2Cbrowser) with name _apim-cs_
```
Comment (petemessina) [inline src/readme.md:5]: Should this be self hosted or azure agent pool?

Comment (petemessina) [reply src/readme.md:5]: If its self-hosted we should document the steps to add the host or link it to the host agent in the scripts.

Comment (seenu433) [reply src/readme.md:5]: @petemessina - This is targeted at the self-hosted agent. We can change it to use the default pool and avoid the step to create a pool

**Review by petemessina — APPROVED** [review]

**Pull request merged**